src/AppBundle/Entity/Client.php

```diff
@@ -47,22 +69,19 @@ class Client
     private $reports;
 
     /**
-     * @var Odr\Odr
+     * @JMS\Type("AppBundle\Entity\Report\Report")
      *
-     * @JMS\Type("AppBundle\Entity\Odr\Odr")
+     * @var array
      */
-    private $odr;
+    private $reportCurrent;
```

Comment:
yes, good point. I wanted to prefix the report-related starting with "report", but it's grammatically incorrect this way. I'll change that, including the API